docs/content/docs/dev/table/concepts/time_attributes.md

```diff
@@ -99,6 +99,105 @@ GROUP BY TUMBLE(time_ltz, INTERVAL '10' MINUTE);
 
 ```
 
+#### Advanced watermark features
+
+In previous versions, many advanced features of watermark (such as watermark alignment) were easy to use through the datastream api, but not so easy to use in sql, so we have extended these features in version 1.18 to enable users to use them in sql as well.
+
+{{< hint warning >}}
+**Note:** Only source connectors that implement the `SupportsWatermarkPushDown` interface (e.g. kafka, pulsar) can use these advanced features. If a source does not implement the `SupportsWatermarkPushDown` interface, but the task is configured with these parameters, the task can run normally, but these parameters will not take effect.
+
+These features all can be configured with dynamic table options or the 'OPTIONS' hint, If the user has configured these feature both in the dynamic table options and in the 'OPTIONS' hint, the options in the 'OPTIONS' hint are preferred. If the user uses 'OPTIONS' hint for the same source table in multiple places, the first hint will be used.
+{{< /hint >}}
+
+##### I. Configure watermark emit strategy
+There are two strategies to emit watermark in flink:
+
+- on-periodic: emit watermark periodic
+- on-event: emit watermark per event data
+
+In the DataStream API, the user can choose the emit strategy through the WatermarkGenerator interface ([Writing WatermarkGenerators]({{< ref "docs/dev/datastream/event-time/generating_watermarks" >}}#writing-watermarkgenerators)). For sql tasks, watermark is emited periodically by default, with a default period of 200ms, which can be changed by the parameter `pipeline.auto-watermark-interval`. If you need to emit watermark per event data, you can configure it in the source table as follows:
+
+```sql
+-- configure in table options
+CREATE TABLE user_actions (
+  ...
+  user_action_time TIMESTAMP(3).
+  WATERMARK FOR user_action_time AS user_action_time - INTERVAL '5' SECOND
+) WITH (
+  'scan.watermark.exit.strategy'='on-event'.
```

Comment:
It should be `scan.watermark.emit.strategy`, and the end of DDL should be `,`.

Reply:
modified